apps/desktop/src/views/SearchView/autoShrinkPolicy.ts

```
@@ -0,0 +1,140 @@
+export interface SearchAutoShrinkEligibilityInput {
+    timedOut: boolean;
+    sessionIdle: boolean;
+    latestCompletedMessageMarker: string | null;
+    latestSeenCompletedMessageMarker: string | null;
+}
+
+function hasPendingToolCalls(message: SessionMessage): boolean {
+    return (
+        message.toolCalls?.some(
+            (toolCall) =>
+                toolCall.status === 'executing' || toolCall.status === 'awaiting_approval'
```

ESLint

[error] Delete `⏎···············`

(prettier/prettier)